docs/resources/User.md

```diff
@@ -33,6 +33,8 @@ There are other rules and restrictions not shared here for the sake of spam and
 | bot?          | boolean   | whether the user belongs to an OAuth2 application                                                    | identify              |
 | system?       | boolean   | whether the user is an Official Discord System user (part of the urgent message system)              | identify              |
 | mfa_enabled?  | boolean   | whether the user has two factor enabled on their account                                             | identify              |
+| banner?       | ?string   | the user's banner, or null if unset                                                                  | identify              |
+| banner_color? | ?number   | the user's banner color encoded as an integer represnetation of hexadecimal color code               | identify              |
```

Comment:
```suggestion
| banner_color? | ?number   | the user's banner color encoded as an integer representation of hexadecimal color code               | identify              |
```

Shouldn't this say its rgb?

Reply:
I just based it off the rope color docs, might be good to change both